python/docs/source/getting_started/installation.rst

```diff
@@ -0,0 +1,120 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+..  Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+============
+Installation
+============
+
+Official releases are available from `the Apache Spark website <https://spark.apache.org/downloads.html>`_.
+Alternatively, you can install it via ``pip`` from PyPI.  PyPI installation is usually for standalone
+locally or as a client to connect to a cluster instead of setting a cluster up.  
+ 
+This page includes the instructions for installing PySpark by using pip, Conda, downloading manually, and building it from the source.
+
+Python Version Supported
+------------------------
+
+Python 3.6 and above.
+
+Using PyPI
+----------
+
+PySpark installation using `PyPI <https://pypi.org/project/pyspark/>`_
+
+.. code-block:: bash
+
+    pip install pyspark
+	
+Using Conda  
+-----------
+
+Conda is an open-source package management and environment management system which is a part of `Anaconda <https://docs.continuum.io/anaconda/>`_ distribution. It is both cross-platform and language agnostic.
+  
+Conda can be used to create a virtual environment from terminal as shown below:
+
+.. code-block:: bash
+
+    conda create -n pyspark_env 
+
+After the virtual environment is created, it should be visible under the list of Conda environments which can be seen using the following command:
+
+.. code-block:: bash
+
+    conda env list
+
+The newly created environment can be accessed using the following command:
+
+.. code-block:: bash
+
+    conda activate pyspark_env
+
+In Conda version earlier than 4.4, the following command might be used:
+
+.. code-block:: bash
+
+    source activate pyspark_env
+
+PySpark installation using ``pip`` under Conda environment is official. 
+
+PySpark can be installed in this newly created environment using PyPI as shown before:
+
+.. code-block:: bash
+
+    pip install pyspark
+
+`PySpark at Conda <https://anaconda.org/conda-forge/pyspark>`_ is not the official release.
```

Comment:
Same, I don't think this matters

Reply:
I would just say, for example:

Note that `PySpark at Conda <https://anaconda.org/conda-forge/pyspark>`_ is available but not necessarily synced with PySpark release cycle because it is maintained by the community separately.